doc/man1/build.info

```diff
@@ -5,25 +5,34 @@ DEPEND[]= \
          openssl-crl.pod \
          openssl-dgst.pod \
          openssl-dhparam.pod \
+         openssl-dsa.pod \
          openssl-dsaparam.pod \
+         openssl-ec.pod \
          openssl-ecparam.pod \
          openssl-enc.pod \
          openssl-gendsa.pod \
+         openssl-genpkey.pod \
          openssl-genrsa.pod \
          openssl-ocsp.pod \
          openssl-passwd.pod \
          openssl-pkcs12.pod \
+         openssl-pkcs7.pod \
          openssl-pkcs8.pod \
+         openssl-pkey.pod \
+         openssl-pkeyparam.pod \
          openssl-pkeyutl.pod \
          openssl-rand.pod \
          openssl-req.pod \
+         openssl-rsa.pod \
          openssl-rsautl.pod \
          openssl-s_client.pod \
          openssl-s_server.pod \
          openssl-s_time.pod \
          openssl-smime.pod \
          openssl-speed.pod \
+         openssl-spkac.pod \
          openssl-srp.pod \
+         openssl-storeutl.pod \
          openssl-ts.pod \
          openssl-verify.pod \
          openssl-x509.pod
```

Comment:
Please split commit 4aa43ff97d93745ae88821ec95f35f1f22807b25 and move the changes in `doc/man1/build.info`

    git show 4aa43ff97d93745ae88821ec95f35f1f22807b25 -- doc/man1/build.info

into a separate commit that states something like

    Generate all templated pod files at build time

in the commit message. See https://github.com/openssl/openssl/pull/10856#issuecomment-576040210 for an explanation.

Reply:
@levitte @richsalz how can we prevent in the future that GENERATE rules for .pod.in files are forgotten in `./doc/man1/build.info` and pod files are generated only at install time? Searching through the build rules
```make
$ find -name build.info -exec grep -n -F .pod.in {} +
./build.info:20:GENERATE[doc/man7/openssl_user_macros.pod]=doc/man7/openssl_user_macros.pod.in
./doc/man1/build.info:41:GENERATE[openssl-ca.pod]=openssl-ca.pod.in
./doc/man1/build.info:43:GENERATE[openssl-cms.pod]=openssl-cms.pod.in
./doc/man1/build.info:45:GENERATE[openssl-crl.pod]=openssl-crl.pod.in
[...]
./doc/man1/build.info:105:GENERATE[openssl-storeutl.pod]=openssl-storeutl.pod.in
./doc/man1/build.info:107:GENERATE[openssl-ts.pod]=openssl-ts.pod.in
./doc/man1/build.info:109:GENERATE[openssl-verify.pod]=openssl-verify.pod.in
./doc/man1/build.info:111:GENERATE[openssl-x509.pod]=openssl-x509.pod.in
./doc/build.info:13:         map { $_ => 1 } glob catfile($sourcedir, "man$section", "*.pod.in");
```

it seems to me like this is the catch-all which triggers the generation at install time:

https://github.com/openssl/openssl/blob/a9396903af941cd598789c8861407df9c24a2fc0/doc/build.info#L12-L13

Wouldn't it be better to remove this catch-all and add a doc-nits check for missing generated files instead?

Reply:
I split things out into the separate commit.  Note that .gitignore has to be manually edited no matter what, so I am less bothered by this than I used to be.  Also, most files are ```.pod.in``` files now:
```
; ls doc/man1/*in | wc
     36      36    1083
; ls doc/man1/*.pod | wc
     56      56    1511
; 
```
If we're going to fix this, then I would rather there be a single "source of truth" and it be driven from .gitignore when it exists.  But I don't care much and there are more important things to worry about :)